rwlib/CMakeLists.txt

```diff
@@ -41,10 +41,9 @@ SET(RWLIB_SOURCES
 	"source/job/WorkContext.cpp"
 	)
 
-add_library(rwlib
+set_source_files_properties(source/gl/gl_core_3_3.c PROPERTIES COMPILE_FLAGS -w)
```

Comment:
Well, other than opening an issue at https://bitbucket.org/alfonse/glloadgen/wiki/Home, I do not really see a solution (except fixing the code by hand).

I think this is the simplest solution and should be fine for now, because this file shouldn't be touched anyway and thus dropping all warnings is harmless.